client/client-multi/client-cpp/kaa/channel/IKaaChannelManager.hpp

```diff
@@ -18,13 +18,27 @@
 #define IKAACHANNELMANAGER_HPP_
 
 #include <list>
+#include <string>
 
 #include "kaa/channel/IDataChannel.hpp"
 #include "kaa/channel/ITransportConnectionInfo.hpp"
 #include "kaa/channel/connectivity/IConnectivityChecker.hpp"
 
 namespace kaa {
 
+/**
+ * The structure represents connection information in readable format.
+ */
+struct EndpointConnectionInfo {
+    std::string endpointIp_;
+    std::string serverIp_;
+
+    EndpointConnectionInfo() = default;
```

Comment:
I left it because of [this|https://github.com/kaaproject/kaa/pull/1013/files#diff-a3f7bfb8dbedef8645fe23657b827651R57].  Though I can replace it with `EndpointConnectionInfo foo("","");`, but this is not quite pretty.